### PR TITLE
Make equation plot with vega-lite

### PR DESCRIPTION
**Pull request by sverhoeven**
Refs #33

### CODE REVIEW

File: run-your-c++-code-on-the-web/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ how to [deal with long running tasks](yetanotherblog).
 So today's aim is to have a simple web app that determines the root of a mathematical function _2x^3 - 4x^2 + 6_, i.e. the
 value of _x_ where _y = 0_.
 
-![equation.svg.png](equation.svg.png)
+![equation.svg](equation.svg)
```
Comment (jspaaks) [inline run-your-c++-code-on-the-web/README.md:22]: does medium support svg?

**Review by jspaaks — APPROVED** [review]

**Comment by jspaaks** [comment]
using my own rule of thumb, this is an improvement and doesnt break anything, hence good to merge.

File: run-your-c++-code-on-the-web/README.md (hunk 1 above)
Comment (sverhoeven) [reply run-your-c++-code-on-the-web/README.md:22]: doh, no it does not.

Comment (sverhoeven) [reply run-your-c++-code-on-the-web/README.md:22]: See https://github.com/NLESC-JCER/run-cpp-on-web/issues/44

**Pull request merged**